InAppMessaging/Example/App/InAppMessaging-Example-iOS/AutoDisplayFlowViewController.m

```diff
@@ -77,6 +78,12 @@ - (IBAction)triggerAnalyticEventTapped:(id)sender {
   [FIRAnalytics logEventWithName:self.eventNameText.text parameters:@{}];
 }
 
+- (IBAction)triggerProgrammaticallyTapped:(id)sender {
```

Comment:
just out of curiosity - is the tapped suffix common in iOS (nothing is necessarily 'tapped' here, since it could be triggered in any number of ways... would triggerProgrammatically be inappropriate naming?

Reply:
In this case it's actually a button being tapped, this is for the test app :)